### PR TITLE
Testimonial Component

### PR DESCRIPTION
**Pull request by Castella1313**
issue solved - #60 
I've made some changes in the size of the image

### CODE REVIEW

**Comment by Castella1313** [comment]
@codewithvk Do check out my PR. I have changed the size of the image, I'm assuming there shouldn't be any problem with bigger screen size too!!

File: frontend/src/Components/Testimonial/Cards.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Cardtemplate from "./Cardtemplate";
+import ocomma from "./../../images/ocomma.png";
+import scomma from "./../../images/scomma.png";
+import andrea from "./../../images/andrea.png";
+
+const details = [{
```
Comment (codewithvk) [inline frontend/src/Components/Testimonial/Cards.js:7]: Make details object as a separate component in `Test_data` folder and fetch from there.

File: frontend/src/Components/Testimonial/Cards.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    description:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ",
+    imgurl:andrea,
+    name:"Andrea Cloe",
+    textcolor:"black",
+    bgcolor:"white"
+},
+]
+
+const ccard = details.map(val =>{
+    console.log(val.comma);
```
Comment (codewithvk) [inline frontend/src/Components/Testimonial/Cards.js:36]: Remove this line

**Review by codewithvk — CHANGES_REQUESTED** [review]
@Castella1313 Make the above changes and attach a screenshot.

File: frontend/public/index 2.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (codewithvk) [inline frontend/public/index 2.html:1]: Why are you create index 2 & 3 file?

**Comment by Castella1313** [comment]
all files have become a mess @codewithvk

**Pull request closed without merging**